simple_api/object/permissions.py

```diff
@@ -7,8 +8,7 @@ def build_permissions_fn(permissions):
     if permissions is None:
         return None
 
-    if not isinstance(permissions, (list, tuple)):
-        permissions = permissions,
+    permissions = ensure_tuple(permissions)
```

Comment:
`ensure_tuple` uz pri zadani vstupu a ten prvni if vymenit za `if not permissions: return None`